cms/envs/common.py

```diff
@@ -904,6 +904,7 @@
 
 SOUTH_MIGRATION_MODULES.update({
     'edx_notifications': 'edx_notifications.stores.sql.migrations',
+    'problem_builder': 'problem_builder.south_migrations',
 })
```

Comment:
@e-kolpakov Good catch - wouldn't it work to rename the `south_migrations` folder to `migrations`? If so, it would be better to not add code specific to mentoring into edx-platform.